ocaml/driver/main_args.ml

```diff
@@ -654,6 +654,12 @@ let mk_match_context_rows f =
   Printf.sprintf
   "<n>  (advanced, see manual section %d.%d.)" chapter section
 
+let mk_parameter f =
+  "-parameter", Arg.String f,
+  "<module name> Registers the imported interface as a parameter for this \n\
+                 open module."
```

Comment:
We could be compiling an interface or implementation, though. How about `<module type name> Compile this module as an open module with <module type name> as a parameter`? (Should probably change the other ones to say `compile` rather than `compiles`)

Reply:
I think a typical OCaml user will interpret "a open module" in a different way. How about:
`<module type name> Compile this module with <module type name> as a parameter`

Reply:
Oh true, we should just ditch the “open module” language altogether already. Fixing here and in the other option descriptions